src/sass/header.sass

```diff
@@ -0,0 +1,13 @@
+.header
+  display: flex
+  justify-content: space-between
+  padding: 32px 80px
+
+  .links
+    display: flex
+    gap: 25px
+
+    a
+      font-family: Montserrat
```

Comment:
This could be defined as the global font

Reply:
Makes sense! Do you mean move this to a Sass variable in my `index.sass`?

Reply:
That file, yes. But not as a variable. Rather style the body so all the text in your app has that family by default

```
body {
  font-family: Montserrat;
}
```

Reply:
Otherwise you have to set the font for every text element 😉